tools/webpack-jumpstart

```diff
@@ -19,14 +19,17 @@ cd "$(realpath -m "$0"/../..)"
 
 wait=''
 rebase=''
+best=''
 while [ $# != 0 ] ; do
     case "$1" in
         --wait)
             wait=1;;
         --rebase)
             rebase=1;;
+        --best)
+            best=1;;
         *)
-            echo "usage: $0 [--rebase] [--wait]" >&2
+            echo "usage: $0 [--best] [--rebase] [--wait]" >&2
```

Comment:
Given that us puny humans are rather sensitive to antimatter, perhaps disallow specifying both options at the same time, at least for the time being?

Reply:
I actually tried it and it works as I expected: it rebases the PR, and then touches the files from your extra commits.  I'll adjust the commit message.